templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+**Describe what this pull request brings**
+<!--
+Some examples:
+
+1. This pull request brings component Hero for the page Home.
+
+2. This pull request adds fixes for the responsive version of the page Blog.
+
+3. This pull request fixes user assignment via phone number in the existing stay.
+-->
+
+**Describe any additional changes**
+<!--
+An example:
+
+- fixes Header's height
+- introduces a new SASS variables $header-height
+- renames global CSS class "block" into "section"
+- fixes a typo in Hero for Home
+
+If this pull request does not contain any additional changes, feel free to remove this section.
+-->
+
+**Steps to test**
+<!--  Need a good example or template -->
+
+**Screenshots**
+<!-- 
+Attach any relevant screenshots.
+
+If this pull request does not represent any visual changes, feel free to remove this section.
+-->
+
+**Checklist**
+
+- [ ] I've tested everything and everything works well
+- [ ] I've checked that the code in this pull request follows [Pixel Point Development Guidelines](https://github.com/pixel-point/development-guidelines)
+- [ ] Yes, I've also checked here, there and there and I've also done this, that and that
```

Comment:
Let's add one last section, `References`, which will hold a number of related PRs/Issues

Reply:
Good idea!